lib/image.ml

```diff
@@ -182,6 +172,12 @@ let has_access_to_portrait conf base p =
      || not (Mutil.contains (sou base img) "/private/"))
 (* TODO: privacy settings should be in db not in url *)
 
+(** [has_access_to_images conf base p] is true iif ???. *)
+let has_access_to_images conf base p =
+  (not conf.no_image)
+  && Util.authorized_age conf base p
+  && Util.is_hide_names conf p
```

Comment:
For `has_access_to_portrait` I would like to have something like:
```ocaml
let has_access_to_portrait conf base p =
   (has_access_to_images conf base p)
   && (
      let img = get_image p in  
      (not (is_empty_string img)) || full_portrait_path conf base p <> None)
```

In `has_access_to_images` I don't think the condition on `Util.is_hide_names` is correct
maybe it should be 
``` &&  (conf.wizard || conf.friend || not ( Util.is_hide_names conf p ) )```
or
``` &&  (conf.wizard || conf.friend || Util.is_public )```
??
I don't know.
Access right in geneweb are a mess.
But it is important to get it right.